hoomd/ParticleGroup.cc

```diff
@@ -710,12 +710,16 @@ void ParticleGroup::thermalizeParticleMomenta(Scalar kT, uint64_t timestep)
                                    access_mode::read);
 
     ArrayHandle<unsigned int> h_tag(m_pdata->getTags(), access_location::host, access_mode::read);
+    ArrayHandle<unsigned int> h_body(m_pdata->getBodies(),
+                                     access_location::host,
+                                     access_mode::readwrite);
```

Comment:
```suggestion
                                     access_mode::read);
```
We're not going to modify the body tag data in this method, so just access it for reading.